.github/workflows/ci.yaml

```diff
@@ -26,6 +26,8 @@ jobs:
         uses: actions/setup-go@v2
         with:
           go-version: "1.14.6"
+      - name: Generate
+        run: make generate
```

Comment:
is the idea there that we just validate that the generate works? Will we need this to run below so that this is available in `crosscompile`? maybe I just don't understand how github actions work?

Reply:
Yes, we want to make sure that our generation tooling works, and the checked-in generated code is up-to-date. It runs before `ci-checks.md`, so that any differences in generated code are marked as a CI failure.